core/src/test/java/com/yandex/yoctodb/util/mutable/stored/V1StoredIndexTest.java

```diff
@@ -25,7 +26,7 @@
     public void buildV1StoredIndex() {
         final String fieldName = "testFiledName";
         V1StoredIndex index = new V1StoredIndex(fieldName);
-        final Map<Integer, UnsignedByteArray> data = initData();
+        final TreeMap<Integer, UnsignedByteArray> data = new TreeMap<>(initData());
```

Comment:
This solution involves extra collection copy. It would be better to replace `HashMap` with `TreeMap` in `initData()`.

Reply:
Oh yes, this is indeed better. I have just made the changes.